### PR TITLE
Translation support

### PR DESCRIPTION
**Pull request by emensch**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist
- [ ] All applicable changes have been documented
- [ ] Any `TO DO` items (or similar) have been entered as GitHub issues and the link to that issue has been included in a comment

### CODE REVIEW

**Review by dkozma — APPROVED** [review]
LGTM!

**Review by dkozma — APPROVED** [review]
Nice use of mixins! LGTM

File: packages/c2pa/src/selectors/selectEditsAndActivity.ts [hunk 1]
```diff
@@ -282,3 +327,22 @@ function translateActionName(
...
+  value1: ValueWithFallback,
+  value2: ValueWithFallback,
+): string | null {
+  return [
+    !value1?.isFallback ? value1?.value : null,
+    !value2?.isFallback ? value2?.value : null,
+    value1?.value ?? null,
+    value2?.value ?? null,
+  ].filter((val) => !!val)[0];
+}
```
Comment (emensch) [inline packages/c2pa/src/selectors/selectEditsAndActivity.ts:358]: Not sure about the implementation here. My brain was frying trying to get the fallback priority stuff worked out and after generating some long & ugly conditional branches this is what I settled on.

Comment (dkozma) [reply packages/c2pa/src/selectors/selectEditsAndActivity.ts:358]: That's fine - the only thing I would request is a docstring on the function that attempts to add a short description of the logic.

**Review by dkozma — APPROVED** [review]
LGTM!

**Pull request merged**